docs/getting_started_with_existing_cluster.md

```diff
@@ -0,0 +1,311 @@
+# Getting Started With An Existing Cluster
```

Comment:
Hi @andrewsykim,

Please don't include this document here. This should be in the CAPI book.

Reply:
Please work with @vincepri as he is working certain file/dir structures so he can build the CAPI book and automatically pull in the assets from the infrastructure providers as long as said assets are in the expected locations.